README.md

```diff
@@ -1,12 +1,13 @@
 # update-pull-request-branch-action
 
-Update a Pull Request from its base branch.
+Update pull requests when their base branches change.
```

Comment:
This isn't true, updating when the base branch changes is your specific use case, but this action is more generic than that

Reply:
Agreed, I've reverted the sentence.